pygmt/helpers/decorators.py

```diff
@@ -60,6 +60,17 @@
             Shift plot origin in y-direction. Full documentation is at
             :gmt-docs:`gmt.html#xy-full`.
          """,
+    "c": r"""panel : int or list
```

Comment:
`panel=True` is also allowed, right?
```suggestion
    "c": r"""panel : int or list or bool
```